CHANGES.txt

```diff
@@ -1,3 +1,8 @@
+Version 0.15.0 (2023-05-19)
```

Comment:
Nit: it's still April :D

Reply:
Good spot!  Link to new docs is here https://deploy-preview-410--snowplow-docs.netlify.app/docs/collecting-data/collecting-from-own-applications/python-tracker/emitters/